src/openvic-simulation/modifier/ModifierManager.cpp

```diff
@@ -1,11 +1,32 @@
 #include "ModifierManager.hpp"
+#include <utility>
+#include "dataloader/NodeTools.hpp"
+#include "modifier/Modifier.hpp"
+#include "modifier/ModifierEffect.hpp"
+
+#define FWD(...) static_cast<decltype(__VA_ARGS__)>(__VA_ARGS__)
 
 using namespace OpenVic;
 using namespace OpenVic::NodeTools;
+using namespace OpenVic::StringUtils;
```

Comment:
Remove this, there's nothing in this file that's dependent on it (and there are so few places using `StringUtils:...` that it's better to be explicit every time)

Reply:
ModifierManager uses `strings_equal_case_insensitive` from StringUtils.